leshan-server-demo/webapp2/src/plugins/axios.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+import Vue from 'vue';
+import axios from "axios";
+
+// Full config:  https://github.com/axios/axios#request-config
+// axios.defaults.baseURL = process.env.baseURL || process.env.apiUrl || '';
+// axios.defaults.headers.common['Authorization'] = AUTH_TOKEN;
+// axios.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded';
```

Comment:
Commented code

Reply:
this is auto generated code done by `vue CLI` I don't know if we should keep it or not. I gues this is just a kind of documentation. 
I will keep it for now (at least to keep it in 1 comment), I don't know if we remove it later ?